src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -203,6 +203,58 @@ List<DarCollection> findDARCollectionByCollectionIds(
   )
   DarCollection findDARCollectionByCollectionId(@Bind("collectionId") Integer collectionId);
 
+  /**
+   * Find the DARCollection and all of its Data Access Requests that has the given collectionId
+   * Instead of only including the latest election for each DAR like findDARCollectionByCollectionId
+   * this query returns all data access elections for each DAR
+   *
+   * @return DarCollection
+   */
+  @RegisterBeanMapper(value = User.class, prefix = "u")
+  @RegisterBeanMapper(value = Institution.class, prefix = "i")
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = DataAccessRequest.class, prefix = "dar")
+  @RegisterBeanMapper(value = Election.class, prefix = "e")
+  @RegisterBeanMapper(value = Vote.class, prefix = "v")
+  @RegisterBeanMapper(value = UserProperty.class, prefix = "up")
+  @RegisterBeanMapper(value = LibraryCard.class, prefix = "lc")
+  @UseRowReducer(DarCollectionReducer.class)
+  @SqlQuery(
+      // nosemgrep
+      "SELECT c.*, "
+          + User.QUERY_FIELDS_WITH_U_PREFIX + QUERY_FIELD_SEPARATOR
+          + Institution.QUERY_FIELDS_WITH_I_PREFIX + QUERY_FIELD_SEPARATOR
+          + UserProperty.QUERY_FIELDS_WITH_UP_PREFIX + QUERY_FIELD_SEPARATOR
+          + LibraryCard.QUERY_FIELDS_WITH_LC_PREFIX + QUERY_FIELD_SEPARATOR
+          + "dd.dataset_id, "
```

Comment:
Suggestion: The query string blocks in the model classes are somewhat problematic. I suggest copying the queries and using a text block. That should help with legibility, debugging, and semgrep warnings.